EKF/terrain_estimator.cpp

```diff
@@ -55,7 +55,10 @@ bool Ekf::initHagl()
 		_terrain_var = sq(_params.range_noise);
 		// success
 		return true;
-
+	} else if (_flow_for_terrain_data_ready) {
+		_terrain_vpos = _state.pos(2);
```

Comment:
This is a guess unless we are on the ground. Might be better to set to height of origin as that is where terrain is most likely to be after takeoff.

Reply:
If on ground, using current position is better.

Reply:
Done